devnets/startNetwork.ts

```diff
@@ -177,7 +181,7 @@ async function spawnChain(
     if (bootnodes) {
       args.push("--bootnodes", bootnodes)
     } else {
-      const { nodeKey, peerId } = await generateNodeKey(binary)
+      const { nodeKey, peerId } = await generateNodeKey(generateNodeKeyBinary)
```

Comment:
The `trappist` binary does not have the utility to generate the node key.

Ideally, we should figure out the algo to generate the node key and not use a binary to compute it.